File: ODsayAndroidSDKSample/src/main/java/app/jw/mapable/gm/Item.java
package app.jw.mapable.gm;

public class Item {

    private String trafficDistance, totalWalk, totalTime, payment, ways;

    private int[] timeValue;




    public String getTrafficDistance(){
        return trafficDistance;
    }

    public String getTotalWalk() {
        return totalWalk;
    }

    public String getTotalTime() {
        return totalTime;
    }

    public String getPayment() {
        return payment;
    }

    public String getWays() {
        return ways;
    }

    public int[] getTimeValue() {
        return timeValue;
    }

    public void setTrafficDistance(String trafficDistance)
    {
        this.trafficDistance = trafficDistance;
    }
    public void setTotalWalk(String totalWalk)
    {
        this.totalWalk = totalWalk;
    }
    public void setTotalTime(String totalTime)
    {
        this.totalTime = totalTime;
    }
    public void setPayment(String payment)
    {
        this.payment = payment;
    }

    public void setWays(String ways) {
        this.ways = ways;
    }

    public void setTimeValue(int[] timeValue) {
        this.timeValue = timeValue;
    }
}

File: app_main/src/main/java/app/jw/mapable/gm/AfterSearch/Item.kt
package app.jw.mapable.gm.AfterSearch

class Item {

    var trafficdistance : String = ""
    var totalwalk = ""
    var totaltime = ""
    var payment1 = ""
    var ways1 = ""


    private lateinit var timeValue: IntArray


    fun getTrafficDistance(): String {
        return trafficdistance
    }

    fun getTotalWalk(): String? {
        return totalwalk
    }

    fun getTotalTime(): String? {
        return totaltime
    }

    fun getPayment(): String? {
        return payment1
    }

    fun getWays(): String? {
        return ways1
    }

    fun getTimeValue(): IntArray {
        return timeValue
    }

    fun setTrafficDistance(trafficDistance: String) {
        trafficdistance = trafficDistance
    }

    fun setTotalWalk(totalWalk: String) {
        totalwalk = totalWalk
    }

    fun setTotalTime(totalTime: String) {
        totaltime = totalTime
    }

    fun setPayment(payment: String) {
        payment1 = payment
    }

    fun setWays(ways: String) {
        ways1 = ways
    }

    fun setTimeValue(timeValue: IntArray) {
        this.timeValue = timeValue
    }
}